ci-operator/step-registry/cucushift/chainupgrade/toimage/cucushift-chainupgrade-toimage-commands.sh

```diff
@@ -414,18 +414,27 @@ function admin_ack() {
     fi
 
     local out; out="$(oc -n openshift-config-managed get configmap admin-gates -o json | jq -r ".data")"
+    echo -e "All admin acks:\n${out}"
     if [[ ${out} != *"ack-4.${SOURCE_MINOR_VERSION}"* ]]; then
         echo "Admin ack not required" && return
     fi
 
     echo "Require admin ack"
     local wait_time_loop_var=0 ack_data
-    ack_data="$(echo ${out} | awk '{print $2}' | cut -f2 -d\")" && echo "Admin ack patch data is: ${ack_data}"
-    oc -n openshift-config patch configmap admin-acks --patch '{"data":{"'"${ack_data}"'": "true"}}' --type=merge
+    ack_data="$(echo ${out} | jq -r "keys[]")" && echo "Admin ack patch data is: ${ack_data}"
+    for ack in ${ack_data};
+    do
+        # e.g.: ack-4.12-kube-1.26-api-removals-in-4.13
+        if [[ "${ack}" == *4\.$TARGET_MINOR_VERSION ]] 
+        then
```

Comment:
A tiny suggestion here, how about move the output here? No need all keys `echo "Admin ack patch data is: ${ack_data}"` in line 424, but ${ack} is the correct one which we care about in output.

Reply:
`echo "Admin ack patch data is: ${ack_data}"` helps us understand how many gates we have, I think it is useful for debug.

and we can also output the exact one. up to you

Reply:
good catch, updated @jiajliu